include/ensmallen_bits/ada_delta/ada_delta.hpp

```diff
@@ -71,7 +71,8 @@ class AdaDelta
            const double epsilon = 1e-6,
            const size_t maxIterations = 100000,
            const double tolerance = 1e-5,
-           const bool shuffle = true);
+           const bool shuffle = true,
+           const bool resetPolicy = true);
```

Comment:
Can you add a description for  the ``resetPolicy``? That applies to the other optimizers as well.

Reply:
Thanks. All done now.